internal/proto/query_coord.proto

```diff
@@ -159,6 +164,17 @@ message GetSegmentInfoResponse {
   repeated SegmentInfo infos = 2;
 }
 
+message RefreshCollectionRequest {
+  common.MsgBase base = 1;
+  int64 collectionID = 2;
```

Comment:
use snake style in proto so that all languages can produce codes with right style, camelCase will make python(or any language uses snake case as default style) code ugly